overviews/core/_posts/2012-09-20-futures.md

```diff
@@ -615,8 +615,42 @@ Invoking the `future` construct uses a global execution context to start an asyn
 Support for extending the Futures API with additional utility methods is planned.
 This will allow external frameworks to provide more specialized utilities.
 
+
 ## Blocking
 
+Futures are generally asynchronous and do not block the underlying execution threads.
+However, in certain cases, it is necessary to block.
+We distinguish two forms of blocking the execution thread:
+invoking arbitrary code that blocks the thread from within the future,
+and blocking from outside another future, waiting until that future gets completed.
+
+
+### Blocking inside a Future
+
+As seen with the global `ExecutionContext`, it is possible to notify an `ExecutionContext` of a blocking call with the `blocking` construct.
+The implementation is however at the complete discretion of the `ExecutionContext`. While some `ExecutionContext` such as `ExecutionContext.global`
+implement `blocking` by means of a `ManagedBlocker`, some execution contexts such as the fixed thread pool: 
+
+    ExecutionContext.fromExecutor(Executors.newFixedThreadPool(x))
+
+will do nothing, as shown in the following:
+
+    implicit val ec = ExecutionContext.fromExecutor(
+                        Executors.newFixedThreadPool(4))
+    Future {
+      // here blocking serves only for documentation purpose
```

Comment:
I don't have a very strong opinion. Maybe we can omit the line which says that it's for documentation, and just say that it informs the EC so that it can optionally adjust its parallelism.